gr-uhd/python/uhd/replaymsgpushbutton.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2023 Ettus Research, a National Instruments Brand.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+#
+
+from PyQt5 import Qt
+from gnuradio import gr
+import pmt
+
+
+class ReplayMsgPushButton(gr.sync_block, Qt.QPushButton):
+    """
+    This block creates a variable push button that creates a message
+    when clicked. The message will be formatted as a dictionary to pass
+    to the RFNoC Replay block
+    """
+
+    def __init__(self, lbl, relBackColor, relFontColor,
+                 command, port, offset=-1, size=-1, time=-1):
+        gr.sync_block.__init__(self, name="ReplayMsgPushButton",
+                               in_sig=None, out_sig=None)
+        Qt.QPushButton.__init__(self, lbl)
+
+        self.lbl = lbl
+
+        self.replayDict = pmt.dict_add(pmt.make_dict(), pmt.intern('command'), pmt.intern(command))
```

Comment:
You can also simply make `self.replayDict` a Python dictionary, which makes the code much more readable.